pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -142,14 +142,14 @@ RUN pip uninstall -y torch torchvision torchaudio torchdata model-archiver multi
 RUN pip install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cpu -U \
     "awscli<2" \
     boto3 \
-    "cryptography>3.2" \
+    cryptography \
     enum-compat==0.0.3 \
-    ipython==8.1.0 \
+    ipython \
     numpy==1.22.2 \
     opencv-python \
     packaging \
     "Pillow>=9.0.0" \
-    "pyyaml>=5.4,<5.5" \
+    pyyaml \
```

Comment:
why unpin these? pyyaml has been known to introduce breaking changes in minor versions, example https://github.com/yaml/pyyaml/issues/265 - if there is no security issue here, we should consider existing user's experience and not leave this completely unpinned

Reply:
pyyaml does have breakage in 6.0 latest 5.x version is 5.4.1. keeping pin.